src/commands.rs

```diff
@@ -88,29 +88,28 @@ enum RusticCmd {
     /// Check the repository
     Check(CheckCmd),
 
-    /// Copy snapshots to other repositories. Note: The target repositories must be given in the config file!
+    /// Copy snapshots to other repositories
     Copy(CopyCmd),
 
-    /// Compare two snapshots/paths
-    /// Note that the exclude options only apply for comparison with a local path
+    /// Compare two snapshots or paths
     Diff(DiffCmd),
 
-    /// dump the contents of a file in a snapshot to stdout
+    /// Dump the contents of a file within a snapshot to the terminal
     Dump(DumpCmd),
 
-    /// Find in given snapshots
+    /// Find patterns in given snapshots
     Find(FindCmd),
 
-    /// Remove snapshots from the repository
+    /// Mark snapshots to be removed from the repository
```

Comment:
this removes snapshot(file)s without marking them.

Reply:
I thought this was just the case when using `--instant-delete`, otherwise `prune` would delete them?

Reply:
Would there be ever snapshots deleted by just running `rustic forget` with never calling `rustic prune`?

Reply:
`forget` removes snapshotfiles directly (unless called with `--dry-run`). And this is basically completely independent from `prune`.
The `--instant-delete` option is only a prune option, for `forget` this is only used if we call `forget --prune` - which is just a shortcut for `rustic forget; rustic prune`.

Reply:
Ah, TIL! Good, reverted it then!